tmt/steps/__init__.py

```diff
@@ -270,7 +275,8 @@ def command(cls):
             commands[method.name] = command
 
         # Create base command with common options using method class
-        method_class = tmt.options.create_method_class(commands)
+        method_class = tmt.options.create_method_class(
+            commands, cls.accept_all_options)
```

Comment:
Brainstorm: Would it make sense to pass here the list of all known supported step methods so that during the `--how` option parsing the check for valid methods could be done and an exception raised when weird method detected? Could this possibly simplify the implementation?